optimade/server/tests/test_server.py

```diff
@@ -442,5 +447,5 @@ def _check_response(self, request, expected_ids, expected_return):
             self.assertEqual(response["meta"]["data_returned"], expected_return)
         except Exception as exc:
             print("Request attempted:")
-            print(f"http://localhost:5000{request}")
+            print(f"http://localhost:5000/optimade{request}")
```

Comment:
We should remove the hardcoded reference to `localhost:5000` here as its misleading, I think just printing `request` is fine.

Reply:
Have put in `self.client.base_url`.